public/locations-list-map.js

```diff
@@ -954,9 +960,31 @@ const MAP_INITIAL_VIEW = {
   us: { zoom: 4, center: { lat: 37.09024, lng: -95.712891 }},
 };
 
+function getMapInitialView() {
+  const url = new URL(window.location.href);
+  const searchParams = new URLSearchParams(url.search);
+  const coords = searchParams.get('coords');
+  // default zoom is pretty tight because if you're passing latlng
+  // you are probably trying to center on a pretty specific location
+  const zoom = parseFloat(searchParams.get('zoom')) || 11;
+  if (coords && coords.length) {
+    const latlng = coords.split(',').map(coord => parseFloat(coord));
```

Comment:
Need to do some more sanitization here. Validate you actually have 2 values, and that lat is in the range [-85..85].